### DISCUSSION TITLE
RIP

### DISCUSSION THREAD

**Discussion by 1ctinus** [benawad/dogehouse#2834] [General]
😭

**Comment by themisir** [comment]
F

**Reply by just1a-person** [reply]
F

**Reply by crxssed7** [reply]
F

**Reply by imansour12** [reply]
> F

F

**Comment by S-codes14** [comment]
· 

> 
    “We'll meet again, Don't know where don't know when"
    “I can’t hold the tears back, You will always be in my heart. R.I.P”
    “Life is pleasant, death is peaceful. It’s the transition that’s troublesome”
    “left yet not forgotten, You memory lives with me forever in my heart”
     “I can’t imagine my life without you. Your memories would live with me forever.” R.I.P

**Reply by nadirabbas** [reply]
this is the most cringiest thing I've seen on GitHub till now ✌

**Reply by S-codes14** [reply]
> this is the most cringiest thing I've seen on GitHub till now v

I know right :)

**Reply by ghost** [reply]
how epic my boy
